src/actions.py

```diff
@@ -0,0 +1,11 @@
+from directions import NORTH, SOUTH, EAST, WEST
+
+"""
+ Actions
+ """
+MOV_N = NORTH
+MOV_E = EAST
+MOV_S = SOUTH
+MOV_W = WEST
```

Comment:
There is no action for "move north", the agent can only way an agent can move is by rotating left or right and moving forward.